WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift

```diff
@@ -8,19 +8,30 @@ public class BlockEditorScreen: ScreenObject {
         $0.navigationBars["Gutenberg Editor Navigation Bar"].buttons["Close"]
     }
 
-    var editorCloseButton: XCUIElement { editorCloseButtonGetter(app) }
-
     let addBlockButtonGetter: (XCUIApplication) -> XCUIElement = {
-        $0.buttons["add-block-button"] // Uses a testID
+        $0.buttons["add-block-button"]
     }
 
-    var addBlockButton: XCUIElement { addBlockButtonGetter(app) }
-
     let moreButtonGetter: (XCUIApplication) -> XCUIElement = {
         $0.buttons["more_post_options"]
     }
 
+    let insertFromUrlButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["Insert from URL"]
+    }
+
+    let applyButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["Apply"]
+    }
+
+    var editorCloseButton: XCUIElement { editorCloseButtonGetter(app) }
+    var addBlockButton: XCUIElement { addBlockButtonGetter(app) }
     var moreButton: XCUIElement { moreButtonGetter(app) }
+    var insertFromUrlButton: XCUIElement { insertFromUrlButtonGetter(app) }
+    var applyButton: XCUIElement { applyButtonGetter(app) }
+
+    var videoUrlPath = "http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4"
+    var audioUrlPath = "http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ElephantsDream.mp4"
```

Comment:
This is a test data. IMO, it better suits the Test file, and can be passed as an argument of `addVideo()` / `addAudio()`.

Reply:
good call out on this one! for text, the test data is on the test file, i updated this in https://github.com/wordpress-mobile/WordPress-iOS/pull/20940/commits/8224bba762a536bcd1287ebaec1691b33f9658a9. i also updated the function name to `addVideoFromUrl(urlPath: videoUrlPath)` / `addAudioFromUrl(urlPath: audioUrlPath)` to be clearer since there are two ways to add videos (one doesn't need a URL).